src/pages/my_page/user_profile/UserOrcid.tsx

```diff
@@ -69,6 +69,7 @@ export const UserOrcid = ({ user }: UserOrcidProps) => {
     queryFn: () => fetchPerson(userCristinId),
     onError: () => dispatch(setNotification({ message: t('feedback.error.get_person'), variant: 'error' })),
   });
+  const fetchCristinPersonRef = useRef(cristinPersonQuery.refetch);
```

Comment:
In short: `useRef` er en wrapper rundt noe som gjør at den ikke trengs som en dependency i `useEffect` lenger. I praksis pakker den det bare inn i ett objekt: `{ current: <VALUE> }`

Reply:
Nice